web/dist/js/sw.js

```diff
@@ -1,14 +1,34 @@
-// version: 2 (improve logos)
+const CACHE_NAME = "biblers-cache-v3";
 
+// Use cache before fetching from the network.
 self.addEventListener("fetch", (e) => {
-    e.respondWith(
-        caches.match(e.request).then(resp => resp || fetch(e.request)),
-    );
+    // Search API requests are not locally cached.
+    if (e.request.url.indexOf("/api/search") > -1) {
+        e.respondWith(
+            caches.match(e.request).then(resp => resp || fetch(e.request)),
+        );
+
+    // Standard pages are cached.
+    } else {
+        e.respondWith(
+            caches.match(e.request).then(initialResp => {
+                console.log(`[Service Worker] Fetching resource: ${e.request.url}`);
```

Comment:
I recommend removing `console.log` in production. This could be removed by a minifier, but that's a bit out of scope right now.

Reply:
Yeah...good call.